### PR TITLE
MSSQL: avoid mentioning anything related to support

### PR DESCRIPTION
**Pull request by zucchinidev**
[#185202516](https://www.pivotaltracker.com/story/show/185202516)

Which other branches should this be merged with (if any)? None

### CODE REVIEW

File: aws-rds-migration.html.md.erb [hunk 1]
```diff
@@ -72,10 +72,7 @@ see [Migrate Data from a Legacy Instance](#steps).
 To migrate data from an existing legacy RDS instance to <%= vars.product_short %>:
 
 1. Create a backup of the legacy RDS instance.
-   1. To create a backup in SQL Server, you need a user with elevated privileges.
-   Credentials associated with applications that use this service do not have the necessary privileges to create a
-   backup. You must contact Support to assist you in the process of obtaining credentials from a user with elevated
-   privileges. Jump to step number 4.
+   1. To create a backup in SQL Server, you need the AWS Master user with elevated privileges. Jump to step number 4.
```
Comment (pivotal-marcela-campo) [inline aws-rds-migration.html.md.erb:75]: Do we need a number here?

**Review by pivotal-marcela-campo — APPROVED** [review]

File: aws-rds-migration.html.md.erb (hunk 1 above)
Comment (zucchinidev) [reply aws-rds-migration.html.md.erb:75]: Great point!

**Comment by HenryBorys** [comment]
@zucchinidev Is the term Master in "AWS Master user" unavoidable? Would "primary" work as a substitute? Just asking, as "Master" is not ideal for inclusive language. But if its unavoidable, then nothing we can do about it. Thanks.

**Comment by zucchinidev** [comment]
Hi @HenryBorys , thanks for asking. 
I think it avoids confusion. It’s exactly the same terminology AWS uses.

See MasterUsername: https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBInstance.html

**Pull request merged**